index.js

```diff
@@ -10,7 +10,7 @@ try {
   const product = input1 * input2;
 
   // Create a custom message
-  const message = `The inputs are ${input1} and ${input2}. Sum is ${sum}, product is ${product}.`;
```

Comment:
The variable `foo` is likely undefined, which will cause an error when trying to concatenate it to the string. Double check if `foo` is supposed to be a different variable name or if it needs to be initialized/defined before being used in the string.